ansible/roles/test/tasks/bgp_neighbor/bgp_neighbor_test_ipv6_bounce_neighbor.yml

```diff
@@ -0,0 +1,49 @@
+# Subtask to bounce an ipv6 neighbor and verify it comes back
```

Comment:
We can avoid another file.
-- Set_fact {{ protocol }} to ipv4\ipv6.
And based on this variable ask for ipv6Unicast.peers  or ipv4Unicast.peers